packages/smithy/goldens/lib/restJson1/lib/src/api_gateway/api_gateway_client.dart

```diff
@@ -31,8 +31,8 @@ class ApiGatewayClient {
 
   final _i2.AWSCredentialsProvider _credentialsProvider;
 
-  _i3.SmithyOperation<_i3.PaginatedResult<_i4.BuiltList<_i5.RestApi>, int>>
-      getRestApis(
+  _i3.SmithyOperation<
+      _i3.PaginatedResult<_i4.BuiltList<_i5.RestApi>, int, String>> getRestApis(
```

Comment:
👍 just saw the analyzer complaining about this spot on my local branch. Thanks for fixing.